crates/subspace-farmer/src/single_disk_farm/farming.rs

```diff
@@ -134,7 +134,8 @@ where
     ) -> Result<
         Vec<(
             SectorIndex,
-            impl ProvableSolutions<Item = Result<Solution<PublicKey>, ProvingError>> + 'a,
+            impl ProvableSolutions<Item = Result<Solution<PublicKey>, ProvingError>>
+            + use<'a, PosTable, Plot>,
```

Comment:
more on the `use` bound: https://blog.rust-lang.org/2024/09/05/impl-trait-capture-rules/